<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphicsDraw</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="GraphicsView" name="graphicsView"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>开始X</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>开始Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>结束X</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>结束Y</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="lineStartXSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="lineStartYSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="lineEndXSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="lineEndYSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>圆心X</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>圆心Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>半径</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>起始角度</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>终止角度</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>方向</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QRadioButton" name="arcClockwiseRadioBtn">
                <property name="text">
                 <string>顺时针</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="arcAnticlockwiseRadioBtn">
                <property name="text">
                 <string>逆时针</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="arcCenterXSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="arcCenterYSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="arcRadiusSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="arcStartAngleSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="arcEndAngleSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>端点1 X</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>端点1 Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>端点2 X</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>端点2 Y</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>控制点X</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>控制点Y</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="markPoint1XSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="markPoint1YSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="markPoint2XSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="markPoint2YSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="markCtrlXSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="markCtrlYSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>位置X</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>位置Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>文字</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="textEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="textPosXSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="textPosYSpinBox">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-999999999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999999999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QSpinBox" name="widthSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>颜色</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="colorLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>线宽</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>缩放</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLine"/>
   <addaction name="actionArc"/>
   <addaction name="actionMark"/>
   <addaction name="actionText"/>
  </widget>
  <action name="actionLine">
   <property name="text">
    <string>直线</string>
   </property>
   <property name="toolTip">
    <string>直线</string>
   </property>
  </action>
  <action name="actionArc">
   <property name="text">
    <string>圆弧</string>
   </property>
   <property name="toolTip">
    <string>圆弧</string>
   </property>
  </action>
  <action name="actionMark">
   <property name="text">
    <string>标注</string>
   </property>
   <property name="toolTip">
    <string>线性标注</string>
   </property>
  </action>
  <action name="actionText">
   <property name="text">
    <string>文字</string>
   </property>
   <property name="toolTip">
    <string>文字</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
